lib/constants.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     TIMESTAMP_TYPE: 'ISO8601',
     SERVICE_HEADER: 'fiware-service',
     SUBSERVICE_HEADER: 'fiware-servicepath',
+    AUTH_HEADER: 'X-Auth-Token',
```

Comment:
Maybe it should be used in lowercase, as the other headers?

Reply:
mmm... not sure, will keystone complain?

Reply:
Goot point. I'm almost sure it will work, but need to be checked. However, let's do it in a separete PR.

In the contet of this PR, a `//FIXME: check Keystone support this in lowercase, then change` would be enough.

Reply:
fix by 871e6ef